examples/main/src/main/java/org/apache/camel/example/StandaloneCamel.java

```diff
@@ -34,29 +34,30 @@ private StandaloneCamel() {
 
     public static void main(String[] args) throws Exception {
         // create a new CamelContext
-        CamelContext camelContext = new DefaultCamelContext();
+        try (CamelContext camelContext = new DefaultCamelContext()) {
```

Comment:
Oh we should avoid this, can you keep the code as before.

Reply:
Ok I will revert this.

Reply:
May I ask, why we ignore that Sonar issue?